infrastructure/modules/aks/law-collection.tf

```
@@ -0,0 +1,54 @@
+resource "azurerm_monitor_data_collection_rule" "law" {
+  name                = "${azurerm_log_analytics_workspace.aks.name}-mdcr"
+  resource_group_name = azurerm_resource_group.aks.name
+  location            = azurerm_resource_group.aks.location
+
+  destinations {
+    log_analytics {
+      workspace_resource_id = azurerm_log_analytics_workspace.aks.id
+      name                  = azurerm_log_analytics_workspace.aks.name
+    }
+  }
+
+  data_flow {
+    streams      = ["Microsoft-ContainerLog", "Microsoft-ContainerLogV2", "Microsoft-KubeEvents", "Microsoft-KubePodInventory"]
+    destinations = ["${azurerm_log_analytics_workspace.aks.name}"]
+  }
+
+  data_flow {
+    streams      = ["Microsoft-Syslog"]
+    destinations = ["${azurerm_log_analytics_workspace.aks.name}"]
+  }
+
+  data_sources {
+    syslog {
+      streams        = ["Microsoft-Syslog"]
+      facility_names = ["auth", "authpriv", "cron", "daemon", "mark", "kern", "local0", "local1", "local2", "local3", "local4", "local5", "local6", "local7", "lpr", "mail", "news", "syslog", "user", "uucp"]
```

Nitpick and personal preference.
Makes it easier, atleast for me, to read.
If this list changes it's also easier to see what has changed
```suggestion
      facility_names = [
        "auth", 
        "authpriv", 
        "cron", 
        "daemon", 
        "mark", 
        "kern", 
        "local0", 
        "local1", 
        "local2", 
        "local3", 
        "local4", 
        "local5", 
        "local6", 
        "local7", 
        "lpr", 
        "mail", 
        "news", 
        "syslog", 
        "user", 
        "uucp",
      ]
```